script/github-actions/codecoverage-format-report.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+
+const codecoverageReport = fs.readFileSync(
```

Comment:
This is really nitpicky, but since it's two words, I would prefer to see `codeCoverage*`.